### ISSUE TITLE
FieldWorks: Test using the designer with VS 2019 16.8

### ISSUE THREAD

**Issue by tombogle** [sillsdev/libpalaso#1032]

**Comment by tombogle** [comment]
@ermshiperete, which branch should this be done on, or does it not matter? I'm not yet seeing any extra csproj files that were added to work with Designer.

**Comment by tombogle** [comment]
I have managed to get 7 projects to build. Can't get most of the solution to build because of this error:
1>IDL Import threw an exception: An entry with the same key already exists.
1>C:\Projects\FieldWorks\Src\ViewsInterfaces\ViewsInterfaces.csproj(54,5): error MSB4181: The "IdlImp" task returned false but did not log an error.
1>Done building project "ViewsInterfaces.csproj" -- FAILED.
I have successfully opened several controls and dialogs in FwCoreDlgs using Designer.
Some that fail include:
ComboTextBox (The base class 'SIL.FieldWorks.FwCoreDlgs.Controls.InnerFwTextBox' could not be  loaded)
DefaultFontsControl (Could not find type 'SIL.FieldWorks.FwCoreDlgs.FwOverrideComboBox')
InnerFwTextBox (The base class 'SIL.FieldWorks.Common.RootSites.SimpleRootSite' could not be  loaded.)
AddCnvrtrDlg (Could not find type 'SIL.FieldWorks.FwCoreDlgs.CnvtrPropertiesCtrl)
CharContextCtrl (Could not find type 'SIL.FieldWorks.FwCoreDlgs.FwPanel'.)
FwWritingSystemSetupDlg (The designer loader did not provide a root component but has not indicated why.)

Probably not worth trying to go any further unless/until I can figure out how to get everything to build.
